### PR TITLE
Comments design

### PR DESCRIPTION
**Pull request by dartcafe**

### CODE REVIEW

File: js/vote.js [hunk 1]
```diff
@@ -144,8 +142,18 @@ $(document).ready(function () {
 			commentBox: comment.value.trim()
 		};
 		$('.new-comment .icon-loading-small').show();
-		$.post(form.action, data, function (data) {
-			$('.comments .comment:first').after('<div class="comment"><div class="comment-header"><span class="comment-date">' + data.date + '</span>' + data.userName + '</div><div class="wordwrap comment-content">' + data.comment + '</div></div>');
+		$.post(form.action, data, function(data) {
+		var newCommentElement = '<li class="comment column"> ' +
```
Comment (splitt3r) [inline js/vote.js]: We should use [Handlebars ](https://docs.nextcloud.com/server/12/developer_manual/app/tutorial.html?highlight=handlebar#creating-a-navigation) here.

Comment (dartcafe) [reply js/vote.js]: Awesome. Was the missing link for me. I will go into it later. 👍

**Comment by dartcafe** [comment]
Little preview.
screenshot-2017-11-11 umfragen - polls

**Comment by v1r0x** [comment]
Looks good 👍 Will have closer look at it the next days.

**Comment by splitt3r** [comment]
👍 the textbox should have the same size and placeholder as the files app one. And perhaps we should make the upper part of the sidebar look more streamlined to the official nextcloud sidebars without the stroke + icons with description as tooltip?

**Comment by dartcafe** [comment]
grafik
screenshot-2017-11-12 umfragen - polls

**Comment by dartcafe** [comment]
????

grafik
grafik

Forced reload?

**Comment by splitt3r** [comment]
Two very small points 😁 how about a bit more space between owner and tags? And a placeholder like 'New comment...' would be great.

**Comment by v1r0x** [comment]
@dartcafe Yes, I always have the dev console open and http cache disabled. But even Ctrl+F5 doesn't help. Still no box.

@splitt3r The placeholder exists, but not visible, because the input field is a div, not a `<textarea>`. Any reason for that @dartcafe ?

**Comment by dartcafe** [comment]
- Added placeholder
- added some margin to the tags

> The placeholder exists, but not visible, because the input field is a div, not a <textarea>. Any reason for that ?

I just adopted the comments app. The div is nicer, because it automatically resizes depending of the content.

Please check https://nc13.yagst.de/index.php/apps/polls/poll/12rdzh9QYiFZaFz4 
I cannot reproduce the issue of the missing text area.

**Comment by v1r0x** [comment]
I pulled the latest commits and now I see the placeholder and can type a comment. But the border is missing.

screenshot-2017-11-15 umfragen - nextcloud

**Comment by dartcafe** [comment]
Aiaiai. Got it. I am checking against NC13. In NC12 this issue occures. 

Please fill out the support form next time. :-) I'll gonna fix this.

**Comment by v1r0x** [comment]
Is there a support form for comments? :wink: 
I'm always on latest stable release. Needs testing as well :stuck_out_tongue:

**Comment by splitt3r** [comment]
Will have a look at it :) If you want to have a look at getting started with vue + nc you could have a look at my [repo](https://github.com/splitt3r/mindmaps/tree/feature/rework-js) or [ChristophWursts](https://github.com/ChristophWurst/nextcloud_vue). It´s far away from ready but got the basic stuff working.

**Comment by splitt3r** [comment]
The home icon seems to be wrong positioned.

Error

**Comment by dartcafe** [comment]
@splitt3r Sure, you pulled the current status of the PR? Your screen shot display the old layout. See my first comments for the new design of the comments...
The problem with the positioning should be fixed (occured in Chrome and Safari).

**Comment by splitt3r** [comment]
Will check that again 😁

**Comment by v1r0x** [comment]
I can see a border! 😁 
One last thing from my side: The cursor doesn't switch to `text`, so it's confusing whether it's a text field or not.

**Comment by dartcafe** [comment]
@splitt3r I don't have this issue neither in oc10, nc11, nc12 nor nc13. ❓ 
I added a border to the exoiry info. 
@v1r0x Which text field?
grafik

**Comment by dartcafe** [comment]
@splitt3r Surprise, surprise. 
Just pulled the latest master of the daily channel (13.0.0 Beta 1), and I have this issue too. Will check that later.

**Comment by dartcafe** [comment]
Fixed the layout for NC13. Still issues in oc10-nc12. Fix it later.

**Comment by v1r0x** [comment]
> @v1r0x Which text field?

The one on the screenshot. For me (on nc12) there is the default cursor, not the text cursor 😕

**Comment by dartcafe** [comment]
Please review. Fixed the controls (breadcrump) and the cursor for the text box. For me, I am ready to merge.

**Comment by splitt3r** [comment]
One small thing left here (Nc 13):

Icon

The icon is left aligned and the number isn´t centered anymore 😁

**Comment by dartcafe** [comment]
@splitt3r Travis is not good to me. :-)  And where is scrutinizer?
BTW @splitt3r and @v1r0x, if you could just close the CRs???

**Comment by splitt3r** [comment]
:joy: disk quota exceeded will habe a look at the Travis builds. What do you mean with CRs?

**Comment by dartcafe** [comment]
CR = Change Request.

**Pull request merged**